packages/accounts-controller/CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [17.2.0]
+
+### Added
+
+- `AccountsController:accountAdded` and `AccountsController:accountRemoved` events added. ([#4496](https://github.com/MetaMask/core/pull/4496))
+- Add and export corresponding event types `AccountsControllerAccountAddedEvent`, `AccountsControllerAccountRemovedEvent`.
+- Export action types `AccountsControllerListMultichainAccounts`,`AccountsControllerGetSelectedMultichainAccount`,`AccountsControllerGetNextAvailableAccountName}Action`. ([#4497](https://github.com/MetaMask/core/pull/4497))
```

Comment:
nit: We generally want top-level changelog entries to be linked to corresponding commits (PRs). How about consolidating these entries e.g.

```suggestion
- Add internal actions and events to `AccountsController` ([#4496](https://github.com/MetaMask/core/pull/4496), [#4497](https://github.com/MetaMask/core/pull/4497))
  - Add events `AccountsController:accountAdded`,  `AccountsController:accountRemoved`, and export corresponding event types `AccountsControllerAccountAddedEvent`, `AccountsControllerAccountRemovedEvent`.
  - Export action types `AccountsControllerListMultichainAccounts`,`AccountsControllerGetSelectedMultichainAccount`,`AccountsControllerGetNextAvailableAccountName}Action`.
```

nit: [Writing in the imperative mood, starting with a present-tense verb](https://common-changelog.org/#241-change) is preferred.